lib/languages.ts

```diff
@@ -165,6 +165,17 @@ const definitions: Record<LanguageKey, LanguageDefinition> = {
         previewFilter: null,
         monacoDisassembly: null,
     },
+    cobol: {
+        name: 'COBOL',
+        monaco: 'cobol',
+        extensions: ['.cob', '.cbl', '.cobol'],
+        alias: [],
+        logoUrl: null, // TODO: Find a better alternative
```

Comment:
Maybe just a plain C (similar to what was used for Pascal)? One could just go with the C from https://commons.wikimedia.org/wiki/File:COBOL_textmark.jpg, for example.